reports/renovate/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM alpine:3.20.0
 
 RUN apk update && apk add curl coreutils yq jq openssl github-cli uuidgen bash
-RUN apk update && apk add python3 py3-pip && pip3 install azure-cosmos pytz
+RUN apk update && apk add python3 py3-pip
+RUN rm /usr/lib/python*/EXTERNALLY-MANAGED && pip3 install azure-cosmos pytz
```

Comment:
have you tried installing with `pipx`?

could also use the flag `--break-system-packages` to pip instead of having to rm the files

More info here: https://stackoverflow.com/a/75821401